dbmongo/js/reduce.algo2/map.ts

```diff
@@ -202,18 +194,19 @@ export function map(this: {
         }
       })
 
-      let output_indexed = output_array.reduce(function (periode, val) {
+      const output_indexed = output_array.reduce(function (periode, val) {
         periode[val.periode.getTime()] = val
         return periode
       }, {} as Record<Periode, SortieMapEntreprise>)
 
       if (v.sirene_ul) {
-        f.sirene_ul(v as DonnéesSireneUL, output_array)
+        f.entr_sirene(v as DonnéesSireneEntreprise, output_array)
```

Comment:
@JazzyPierrot Es-tu d'accord pour que je passe directement `v.sirene_ul` au lieu de `v`, ici ? (dans une prochaine PR)

Ça éviterait l'usage d'un cast.

Reply:
oui c'est plus logique

Reply:
ok c'est noté ! je ferai ça dans une PR séparée